core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1753,7 +1753,7 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Unpersist an RDD from memory and/or disk storage
    */
-  private[spark] def unpersistRDD(rddId: Int, blocking: Boolean = true) {
+  private[spark] def unpersistRDD(rddId: Int, blocking: Boolean) {
```

Comment:
This is a private method and its optional arg value is always specified, so, I removed the default to avoid confusion.